src/metabase/query_processor/card.clj

```diff
@@ -38,11 +38,10 @@
 
 (defn- cache-strategy
   [card dashboard-id]
-  (when (public-settings/enable-query-caching)
-    (or (granular-cache-strategy card dashboard-id)
-        {:type            :ttl
-         :multiplier      (public-settings/query-caching-ttl-ratio)
-         :min_duration_ms (long (* (public-settings/query-caching-min-ttl) 1000))})))
+  (or (granular-cache-strategy card dashboard-id)
+      (some-> (t2/select-one :model/CacheConfig :model "root" :model_id 0 :strategy :ttl)
+              api.cache/row->config)))
```

Comment:
It would be nice to encapsulate this low level implementation detail in somewhere like `api/cache.clj` rather.

Reply:
why not in `granular-cache-strategy` itself?

Reply:
actually I don't think this is needed. `granular-cache-strategy` will is getting the root config already.

Reply:
Well OSS one will still be in this file :)

Reply:
right, in that case can we implement `granular-cache-strategy` for OSS to return the root config?

Reply:
done